cmd/jujud/agent/util_test.go

```diff
@@ -67,10 +67,7 @@ func (s *commonMachineSuite) SetUpSuite(c *gc.C) {
 	s.AgentSuite.SetUpSuite(c)
 	s.PatchValue(&jujuversion.Current, coretesting.FakeVersionNumber)
 	s.PatchValue(&stateWorkerDialOpts, mongotest.DialOpts())
-}
-
-func (s *commonMachineSuite) TearDownSuite(c *gc.C) {
-	s.AgentSuite.TearDownSuite(c)
```

Comment:
just wondering why we remove teardown suite.

Reply:
It's a redundant call. If there is nothing in the TearDown but the call to super.TearDown, the infrastructure will call it anyway.